project/src/components/pages/add-review/add-review.jsx

```diff
@@ -1,191 +1,65 @@
 import React from 'react';
-import {Link} from 'react-router-dom';
+import { Link, useParams } from 'react-router-dom';
 import Logo from '../../logo/logo';
-import PageFooter from '../../page-footer/page-footer';
-import FilmList from '../../film-list/film-list';
-import filmListProp from '../../film-list/film-list.prop';
+import ReviewForm from '../../review-form/review-form';
+import {filmListProp} from '../../film-list/film-list.prop';
 
 
-function AddReview({filmList}) {
+function AddReview({films}) {
   const url = '#';
+  const filmId = useParams();
+  const currentFilm = films.find((film) => film.id === filmId.id);
 
   return (
-    <>
-      <section className="film-card film-card--full">
-        <div className="film-card__hero">
-          <div className="film-card__bg">
-            <img src="img/bg-the-grand-budapest-hotel.jpg" alt="The Grand Budapest Hotel" />
-          </div>
+    <section className="film-card film-card--full">
+      <div className="film-card__header">
+        <div className="film-card__bg">
+          <img src={currentFilm.backgroundImage} alt={currentFilm.name} />
+        </div>
 
-          <h1 className="visually-hidden">WTW</h1>
+        <h1 className="visually-hidden">WTW</h1>
 
-          <header className="page-header film-card__head">
-            <Logo />
+        <header className="page-header">
+          <Logo />
 
-            <ul className="user-block">
-              <li className="user-block__item">
-                <div className="user-block__avatar">
-                  <img src="img/avatar.jpg" alt="User avatar" width="63" height="63" />
-                </div>
+          <nav className="breadcrumbs">
+            <ul className="breadcrumbs__list">
+              <li className="breadcrumbs__item">
+                <Link to={`/films/${filmId.id}/`}  className="breadcrumbs__link">{currentFilm.name}</Link>
```

Comment:
слэш лишний, в роутах его нет

```suggestion
                <Link to={`/films/${filmId.id}`}  className="breadcrumbs__link">{currentFilm.name}</Link>
```